Tasks/AzureResourceGroupDeploymentV2/operations/DeploymentGroupExtensionHelper.ts

```diff
@@ -30,9 +29,15 @@ export class DeploymentGroupExtensionHelper {
         var listOfVms: az.VM[] = await this.azureUtils.getVMDetails();
         var extensionAddedOnVMsPromises: Promise<any>[] = [];
         for (var vm of listOfVms) {
-            extensionAddedOnVMsPromises.push(this.addExtensionOnSingleVM(vm));
+            extensionAddedOnVMsPromises.push(this.addExtensionOnSingleVM(vm).catch(err => err));
```

Comment:
Consider adding VM name before the error message.

Reply:
I see the message will come correctly from below. Should you rather add the VM name prefix here so that it covers all cases

Reply:
Done